qa/standalone/test_ceph_daemon.sh

```diff
@@ -154,7 +160,27 @@ $SUDO $CEPH_DAEMON shell --fsid $FSID --config $CONFIG --keyring $KEYRING -- \
 # WRITE ME
 
 ## adopt
-# WRITE ME
+for tarball in $TEST_TARS; do
+    TMP_TAR_DIR=`mktemp -d -p $TMPDIR`
+    tar xzvf $tarball -C $TMP_TAR_DIR
+    for t in mon mgr mds; do
+        $SUDO $CEPH_DAEMON --image $IMAGE_MASTER adopt \
+                           --style legacy \
+                           --name $t.admin \
+                           --legacy-dir $TMP_TAR_DIR
```

Comment:
If i'm following here, this assumes that each of these tar files includes mon.admin, mgr.admin, and mds.admin.

I wonder if we should put a .txt next to each tgz that has a list of the entities that are included.  Then this could loop over tar files, extract to a tmpdir, and then loop over expected entities and import.  We could even adjust ceph-daemon ls to take --legacy-dir too and ensure that the old daemons show up in the listing?  That would let us add test cases without modifying this file.  (Or maybe that's more trouble than it's worth and we should just hard-code the test cases here, skipping the glob entirely...)

Reply:
BTW the test could also finish with a ceph-daemon ls that verifies that the adopted entity(ies) are now listed as style 'ceph-daemon:v1'

Reply:
Instead of a .txt file, perhaps we can just query the list of `legacy` entities using #31585? 

It's less explicit, but allows for the tarball to be changed without needing to also update a .txt...

Reply:
Yeah that is generally fine, unless there is a bug in the listing portion, which would mean we never test importing a particular daemon and wouldn't notice.  No strong feelings either way here.